lib/archethic/p2p/supervisor.ex

```diff
@@ -11,11 +11,20 @@ defmodule Archethic.P2P.Supervisor do
 
   use Supervisor
 
+  @table_name :connection_status
```

Comment:
Maybe you should use a function in the Connection module to give the name of the table.